Pyjion/absint.cpp

```diff
@@ -1616,6 +1616,39 @@ void AbstractInterpreter::build_set(size_t argCnt) {
     inc_stack();
 }
 
+void AbstractInterpreter::extend_set_recursively(Local setTmp, size_t argCnt) {
+    if (argCnt == 0) {
+        return;
+    }
+
+    auto valueTmp = m_comp->emit_define_local();
```

Comment:
Can't you reverse the order of the parameters on PyJit_UpdateSet and avoid using the temp here and just do the emit_load_local(setTmp) and consume the value that's already on the stack?